hadoop-ozone/dist/src/main/smoketest/s3/commonawslib.robot

```diff
@@ -141,7 +143,8 @@ Setup s3 tests
     Run Keyword        Generate random prefix
     Run Keyword        Install aws cli
     Run Keyword if    '${OZONE_S3_SET_CREDENTIALS}' == 'true'    Setup v4 headers
-    Run Keyword if    '${BUCKET}' == 'generated'            Create generated bucket    ${BUCKET_LAYOUT}
+    Run Keyword if    '${BUCKET}' == 'generated'            Create generated bucket with OBS   ${BUCKET_LAYOUT}
+    Run Keyword if    '${BUCKET1}' == 'generated'           Create generated bucket with FSO   ${BUCKET_LAYOUT1}
```

Comment:
Referring to the below comment, the BUCKET_LAYOUT variables wouldn't need to be passed here.